Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/PDFDocumentLayout.mm

```diff
@@ -147,12 +174,21 @@
 
 WebCore::FloatRect PDFDocumentLayout::boundsForPageAtIndex(PageIndex index) const
 {
-    if (index >= m_pageBounds.size())
+    if (index >= m_pageGeometry.size())
         return { };
 
-    return m_pageBounds[index];
+    return m_pageGeometry[index].normalizedBounds;
 }
 
+int PDFDocumentLayout::rotationForPageAtIndex(PageIndex index) const
```

Comment:
Doesn't look like this method is called anywhere – is that for a subsequent patch?